packages/storybook-addon-designs/package.json

```diff
@@ -23,24 +23,26 @@
   },
   "devDependencies": {
     "@figspec/components": "^1.0.0",
-    "@storybook/addon-docs": "6.5.9",
-    "@storybook/addons": "6.5.9",
-    "@storybook/api": "6.5.9",
-    "@storybook/client-api": "6.5.9",
-    "@storybook/components": "6.5.9",
-    "@storybook/core-events": "6.5.9",
-    "@storybook/theming": "6.5.9",
+    "@storybook/addon-docs": "7.0.7",
+    "@storybook/components": "7.0.7",
+    "@storybook/core-events": "7.0.7",
+    "@storybook/manager-api": "^7.0.7",
+    "@storybook/preview-api": "^7.0.7",
+    "@storybook/theming": "7.0.7",
     "@types/react": "^17.0.0",
     "@types/react-dom": "^17.0.0",
     "@types/webpack-env": "^1.13.9",
     "figma-js": "^1.13.0",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
     "typescript": "^4.7.3"
   },
   "peerDependencies": {
     "@storybook/addon-docs": "^6.4.0 || ^7.0.0",
     "@storybook/addons": "^6.4.0 || ^7.0.0",
-    "@storybook/api": "^6.4.0 || ^7.0.0",
     "@storybook/components": "^6.4.0 || ^7.0.0",
+    "@storybook/manager-api": "^7.0.0",
+    "@storybook/preview-api": "^7.0.0",
```

Comment:
This change removes v6.4 compatibility, right? If this is required we should drop support for v6.4 (peerDeps and README)

Reply:
that's right ☝️